doc/templates/body.php

```diff
@@ -0,0 +1,7 @@
+<body>
+<div class="container-fluid">
+    <?php echo $this->render('core'); ?>
+</div>
+<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/8.6/highlight.min.js"></script>
```

Comment:
Use schema less URIs require to visualize documentation using a webserver, even for visualize the generated static HTML.

Preferable to use `https` always